ansible_base/lib/utils/models.py

```diff
@@ -42,3 +43,17 @@ class AuthToken(BaseModel):
     """
     setattr(relation, '__prevent_search__', True)
     return relation
+
+
+def user_summary_fields(user):
```

Comment:
Does this need to be a separate method here?

Reply:
separate from what? This is the method we attach to `AUTH_USER_MODEL` (if needed), I don't see how that can be reduced.

Reply:
I was thinking merge this logic into decorate_user_model since its only called from there. i.e. build the `sf` variable in `decorate_user_model` and pass that into the `add_to_class` call there.

Reply:
That doesn't make sense to me because `sf` is computed from the function argument `user` object. `decorate_user_model` doesn't have the user properties needed for it.